Month3/LinkedList/LinkedList/TStack.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+#include "TSLList.h"
+template <typename T>
+
```

Comment:
I wnat make undesrtandable

Reply:
Then it would be best to separate with aan empty line the includes to the template, but template usually goes along without any space with whatever structure it is a template of